src/drivers/utils/opfs-utils.ts

```diff
@@ -0,0 +1,394 @@
+import { createError } from ".";
+
+export const DRIVER_NAME = "opfs";
+
+function ignoreNotfoundError(error: any): null {
+  if (error.name === "NotFoundError") return null;
+  throw error;
+}
+
+/**
+ * Normalize a path, removing empty segments and leading/trailing slashes
+ */
+export function normalizePath(path: string): string {
+  const normalizedWrappedPath = `/${path}/`
+    // Replace colons with slashes
+    .replace(/:/g, "/")
+
+    // Remove . segments
+    .replace(/\/\.\//g, "/")
+
+    // Remove duplicate slashes
+    .replace(/\/{2,}/g, "/");
```

Comment:
Perf: Can we merge these into one regex?

Reply:
I generally prefer code clarity over micro optimizations, but I'm also not too well-versed regarding performance impact of regexes, so you're the boss of course. 😁 
Reduced normalization to one single regex.